server/api/openapi.yaml

```diff
@@ -584,3 +638,12 @@ components:
       - name
       - namespace
       type: object
+    inline_object:
```

Comment:
inlineApiObject ?

Reply:
you are right this is horrible i will have this file removed